### DISCUSSION TITLE
Slow uploads?

### DISCUSSION THREAD

**Discussion by jdwires** [Natizyskunk/vscode-sftp#458] [General]
I've seen other people with a similar issue, but haven't found an actual fix. Every upload of even a 1kb file takes at least 10 seconds, but uploading to the same server with an external FTP client will take a fraction of a second. The debug log says it's using passive mode. 

Any thoughts?

**Comment by Natizyskunk** [comment]
@jdwires,
I'm currently looking at a potentiel remodel to use active mode instead but don't know for now when it'll be released.
